### ISSUE TITLE
Module accessibility tests fail on fresh clone

### ISSUE THREAD

**Issue by XaserAcheron** [felixrieseberg/ember-electron#313]
I just did a fresh fork+clone of the `ember-electron` repo, and a couple of the local tests are failing:

```
ok 1 PhantomJS 2.1 - ESLint | app: app.js
ok 2 PhantomJS 2.1 - ESLint | app: resolver.js
ok 3 PhantomJS 2.1 - ESLint | app: router.js
ok 4 PhantomJS 2.1 - ESLint | tests: helpers/destroy-app.js
ok 5 PhantomJS 2.1 - ESLint | tests: helpers/module-for-acceptance.js
ok 6 PhantomJS 2.1 - ESLint | tests: helpers/resolver.js
ok 7 PhantomJS 2.1 - ESLint | tests: helpers/start-app.js
ok 8 PhantomJS 2.1 - ESLint | tests: test-helper.js
ok 9 PhantomJS 2.1 - ESLint | tests: unit/end-to-end-test.js
ok 10 PhantomJS 2.1 - ember-electron end-to-end test: ember is available
not ok 11 PhantomJS 2.1 - ember-electron end-to-end test: node modules are accessible
    ---
        actual: >
            null
        expected: >
            null
        stack: >
            http://localhost:7357/assets/tests.js:158:35
            runTest@http://localhost:7357/assets/test-support.js:3323:34
            run@http://localhost:7357/assets/test-support.js:3309:13
            http://localhost:7357/assets/test-support.js:3515:15
            advance@http://localhost:7357/assets/test-support.js:2968:26
            begin@http://localhost:7357/assets/test-support.js:4780:27
            http://localhost:7357/assets/test-support.js:4740:11
        message: >
            Died on test #1 http://localhost:7357/assets/tests.js:157:19
            exports@http://localhost:7357/assets/vendor.js:111:37
            requireModule@http://localhost:7357/assets/vendor.js:32:25
            http://localhost:7357/assets/test-support.js:7366:18
            require@http://localhost:7357/assets/test-support.js:7356:32
            loadModules@http://localhost:7357/assets/test-support.js:7348:23
            loadTests@http://localhost:7357/assets/test-support.js:7193:33
            start@http://localhost:7357/assets/test-support.js:7228:16
            http://localhost:7357/assets/tests.js:107:28
            exports@http://localhost:7357/assets/vendor.js:111:37
            requireModule@http://localhost:7357/assets/vendor.js:32:25
            global code@http://localhost:7357/assets/tests.js:170:8: Can't find variable: requireNode
        Log: |
    ...
not ok 12 PhantomJS 2.1 - ember-electron end-to-end test: local modules are accessible
    ---
        actual: >
            null
        expected: >
            null
        stack: >
            http://localhost:7357/assets/tests.js:166:29
            runTest@http://localhost:7357/assets/test-support.js:3323:34
            run@http://localhost:7357/assets/test-support.js:3309:13
            http://localhost:7357/assets/test-support.js:3515:15
            advance@http://localhost:7357/assets/test-support.js:2968:26
            begin@http://localhost:7357/assets/test-support.js:4780:27
            http://localhost:7357/assets/test-support.js:4740:11
        message: >
            Died on test #1 http://localhost:7357/assets/tests.js:164:19
            exports@http://localhost:7357/assets/vendor.js:111:37
            requireModule@http://localhost:7357/assets/vendor.js:32:25
            http://localhost:7357/assets/test-support.js:7366:18
            require@http://localhost:7357/assets/test-support.js:7356:32
            loadModules@http://localhost:7357/assets/test-support.js:7348:23
            loadTests@http://localhost:7357/assets/test-support.js:7193:33
            start@http://localhost:7357/assets/test-support.js:7228:16
            http://localhost:7357/assets/tests.js:107:28
            exports@http://localhost:7357/assets/vendor.js:111:37
            requireModule@http://localhost:7357/assets/vendor.js:32:25
            global code@http://localhost:7357/assets/tests.js:170:8: Can't find variable: requireNode
        Log: |
    ...

1..12
# tests 12
# pass  10
# skip  0
# fail  2
Testem finished with non-zero exit code. Tests failed.
```

I just ran `npm install` followed by `ember test`, nothing fancy/weird. I'm on Windows 8.1 though, in case _that_ is weird/relevant. :P

**Comment by bendemboski** [comment]
There's not really anything I can do with that minified stack trace. Can you try running `ember serve` and then visiting `/test` in Chrome? And if so, capture a stack trace with source maps?
